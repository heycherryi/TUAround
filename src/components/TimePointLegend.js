import { faCalendarDay } from '@fortawesome/free-solid-svg-icons';
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
import React from 'react';

function TimePointLegend() {
  return (
    <div className="small">
      <ul className="list-inline">
        <li className="list-inline-item mb-2"><span className="px-1 border">0:00 AM</span> Scheduled time.</li>
        <li className="list-inline-item mb-2"><strike className="text-muted px-1 border">0:00 AM</strike> Previous scheduled time.</li>
        <li className="list-inline-item mb-2"><strong className="text-primary px-1 border">0:00 AM</strong> Updated time from tracking.</li>
        <li className="list-inline-item mb-2"><strong className="px-1 border"><FontAwesomeIcon icon={faCalendarDay} fixedWidth={true} className="px-3" /></strong> Time spans into the next day.</li>
      </ul>
    </div>
  );
}

export default TimePointLegend;
