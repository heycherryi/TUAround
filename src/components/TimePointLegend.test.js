/* globals test, expect */

import React from 'react';
import { render } from '@testing-library/react';
import TimePointLegend from './TimePointLegend';

test('renders TimePointLegend', () => {
  const { container } = render(
    <TimePointLegend />,
  );
  expect(container).toMatchSnapshot();
});
