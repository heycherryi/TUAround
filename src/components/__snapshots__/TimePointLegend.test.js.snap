// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders TimePointLegend 1`] = `
<div>
  <div
    class="small"
  >
    <ul
      class="list-inline"
    >
      <li
        class="list-inline-item mb-2"
      >
        <span
          class="px-1 border"
        >
          0:00 AM
        </span>
         Scheduled time.
      </li>
      <li
        class="list-inline-item mb-2"
      >
        <strike
          class="text-muted px-1 border"
        >
          0:00 AM
        </strike>
         Previous scheduled time.
      </li>
      <li
        class="list-inline-item mb-2"
      >
        <strong
          class="text-primary px-1 border"
        >
          0:00 AM
        </strong>
         Updated time from tracking.
      </li>
      <li
        class="list-inline-item mb-2"
      >
        <strong
          class="px-1 border"
        >
          <svg
            aria-hidden="true"
            class="svg-inline--fa fa-calendar-day fa-fw px-3"
            data-icon="calendar-day"
            data-prefix="fas"
            focusable="false"
            role="img"
            viewBox="0 0 448 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M128 0c17.7 0 32 14.3 32 32V64H288V32c0-17.7 14.3-32 32-32s32 14.3 32 32V64h48c26.5 0 48 21.5 48 48v48H0V112C0 85.5 21.5 64 48 64H96V32c0-17.7 14.3-32 32-32zM0 192H448V464c0 26.5-21.5 48-48 48H48c-26.5 0-48-21.5-48-48V192zm80 64c-8.8 0-16 7.2-16 16v96c0 8.8 7.2 16 16 16h96c8.8 0 16-7.2 16-16V272c0-8.8-7.2-16-16-16H80z"
              fill="currentColor"
            />
          </svg>
        </strong>
         Time spans into the next day.
      </li>
    </ul>
  </div>
</div>
`;
